src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -466,8 +468,11 @@ void mongoSubCacheUpdate(const std::string& tenant, const std::string& subId, lo
 
   if (lastNotificationTime != 0)
   {
-    // Update lastNotificationTime
-    condition = BSON("_id" << OID(subId) << CSUB_LASTNOTIFICATION << BSON("$lt" << lastNotificationTime));
+    // Update lastNotificationTime    
+    condition = BSON("_id" << OID(subId) << "$or" << BSON_ARRAY(
```

Comment:
This is the fix for:

"Fix: csub cache sync logic not persisting lastNotification when the field doesn't initially exist in DB"